cmd/terramate/cli/cli.go

```diff
@@ -1083,7 +1083,7 @@ func (c *cli) printMetadata() {
 
 	// TODO(katcipis): we need to print other project metadata too.
 	c.log("\nproject metadata:")
-	c.log("\tterramate.stacks.all=%v", projmeta.Stacks)
+	c.log("\tterramate.stacks.all=%v", projmeta.Stacks())
```

Comment:
uh what is `stacks.all`? ;) isn't this `stacks.list`?

Reply:
will have to fix it in a separate PR because tests are also looking for `.all` :laughing: